### PR TITLE
Integrate bugsnag on sample app

### PR DESCRIPTION
**Pull request by waelba**
- Add bugsnag to sample app

### CODE REVIEW

**Review by DrissTM — APPROVED** [review]
It's working great so good to me :+1:

**Comment by jarthod** [comment]
Same thing than in the other PR, please avoid the 300 files

**Comment by waelba** [comment]
OK

**Comment by jarthod** [comment]
Shouldn't the cocoapod install folder be outside of the repository? it's like the bundler directory isn't it?

**Comment by waelba** [comment]
It will be installed directly because of our script that install it when compiling source: https://github.com/dimelo/Dimelo-iOS-Source/blob/4bc785fe972a2751e5348b7306828d3c85300f36/build_scripts/update_sample_app.sh#L45

**Comment by jarthod** [comment]
Yes but what I'm saying is that it probably shouldn't be in the repository, it doesn't serve any purpose here, just adding noise. Shouldn't it be gitignored? @DrissTM ?

**Comment by DrissTM** [comment]
I think we should remove the sampleapp compilation from the script, IMO it doesn't make any sense to compile the sample in our repo, anyone that wants to use our app should clone the repo and compile it themselves, WDYT ?

**Comment by waelba** [comment]
Ok I will do that

File: .gitignore [hunk 1]
```diff
@@ -13,4 +13,5 @@ DerivedData/
 *.mode2v3
 *.perspectivev3
 *.moved-aside
-
+CocoaPods\ Install/Pods/
+CocoaPods\ Install/Podfile.lock
```
Comment (jarthod) [inline .gitignore]: I think the `Podfile.lock` file should stay in git, as this is the one which guarantee we're using the same version. Just the `Pods` folder should be gone. This is the same with `bundler`.

Comment (waelba) [reply .gitignore]: when running pod install on client side, Podfile.lock will be created automatically

Comment (jarthod) [reply .gitignore]: Yes I know, but it may contain different versions

Comment (waelba) [reply .gitignore]: OK

Comment (jarthod) [reply .gitignore]: You need to commit the .lock file again, and make sure it's the last version :)

Comment (waelba) [reply .gitignore]: YES

Comment (jarthod) [reply .gitignore]: The Podfile.lock is missing bugsnag ^^

Comment (waelba) [reply .gitignore]: fixed

**Review by jarthod — APPROVED** [review]

**Pull request merged**